tests/group_tree.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import sunbeam
+
+
+class TestGroupTree(unittest.TestCase):
+    def setUp(self):
+        norne = '../../examples/data/norne/NORNE_ATW2013.DATA'
+        self.norne = sunbeam.parse(norne, ('PARSE_RANDOM_SLASH', sunbeam.action.ignore))
+
+    def test_group_obj(self):
+        self.gr_obj = self.norne.schedule.group_tree(0)
+        parent = self.gr_obj._parent('PROD')
+        children = self.gr_obj._children('PROD')
+
+        self.assertEqual('FIELD', parent)
+        self.assertEqual('MANI-B1', children[0])
+        self.assertEqual(6, len(children))
+
+    def test_py_struct(self):
+        tree = self.norne.schedule.gtree(0)
+        parent = tree['MANI-B1']['parent']
+        children = tree['MANI-B1']['children']
```

Comment:
> `children = tree['MANI-B1']['children']`

I'm not sure that I would want to write `xxx.['children']` instead of `xxx.children`?